File: typescript/libs/system/workspace/adapters-json/src/lib/database/invoke/invoke-result.ts
import { IInvokeResult, ISession } from '@allors/system/workspace/domain';
import { InvokeResponse } from '@allors/database/system/protocol-json';
import { Result } from '../result';

export class InvokeResult extends Result implements IInvokeResult {
  constructor(session: ISession, invokeResponse: InvokeResponse) {
    super(session, invokeResponse);
  }
}

File: dotnet/System/Workspace/Adapters/Allors.Workspace.Adapters.Json/Database/Invoke/InvokeResult.cs
// <copyright file="RemoteInvokeResult.cs" company="Allors bv">
// Copyright (c) Allors bv. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Workspace.Adapters.Json
{
    using Allors.Protocol.Json.Api.Invoke;

    public class InvokeResult : Result, IInvokeResult
    {
        public InvokeResult(IWorkspace workspace, InvokeResponse invokeResponse) : base(workspace, invokeResponse)
        {
        }
    }
}
